### PR TITLE
Added cred helper info

### PR DESCRIPTION
**Pull request by grantslone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1435,6 +1435,31 @@ $ git config --global credential.helper cache
 $ git config --global credential.helper 'cache --timeout=3600'
 # Set the cache to timeout after 1 hour (setting is in seconds)
 ```
+Find a credential helper.
```
Comment (RichardLitt) [inline README.md]: Change to: "To find a credential helper:"

File: README.md [hunk 2]
```diff
@@ -1435,6 +1435,31 @@ $ git config --global credential.helper cache
...
+$ git config --global credential.helper manager
+# Git for Windows 2.7.3+
+```
+
+```sh
+$ git config --global credential.helper gnome-keyring
+# Ubuntu and other GNOME-based distros
+```
+
+More credential helpers can be found for different distributions/OS's
```
Comment (RichardLitt) [inline README.md]: Add a period

File: README.md [hunk 3]
```diff
@@ -1435,6 +1435,31 @@ $ git config --global credential.helper cache
...
 # Set the cache to timeout after 1 hour (setting is in seconds)
 ```
+Find a credential helper.
+
+```sh
+$ git help -a | grep credential
+# Shows you possible credential helpers
+```
+
+For OS specific credential caching.
```
Comment (RichardLitt) [inline README.md]: Change . to a :

**Comment by RichardLitt** [comment]
Ping @grantslone. Do you want to make these changes?

**Comment by grantslone** [comment]
Looks good!

**Comment by RichardLitt** [comment]
@grantslone Why did you resolve those? They weren't edited. O.o

**Comment by RichardLitt** [comment]
Thanks, merged.

**Pull request merged**